src/Command.php

```diff
@@ -162,7 +162,7 @@ public function getExecCommand()
     {
         if ($this->_execCommand===null) {
             $command = $this->getCommand();
-            if (!$command) {
+            if (!$command || !is_executable($command)) {
```

Comment:
Yes, sorry for the confusion. But I still think, we don't need a check (see my other comment).